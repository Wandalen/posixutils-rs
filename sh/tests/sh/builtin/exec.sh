(exec echo hello)
(exec -- echo hello)

(exec 1> $TEST_WRITE_DIR/exec.txt; echo $?; echo hello)

cat $TEST_WRITE_DIR/exec.txt
rm $TEST_WRITE_DIR/exec.txt

exec echo hello

echo wrong
