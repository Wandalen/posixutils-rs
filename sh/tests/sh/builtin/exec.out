hello
hello
0
hello
hello
